stable/redis/values.yaml

```diff
@@ -341,6 +341,17 @@ metrics:
   # podAnnotations: {}
   # podLabels: {}
 
+  # Enable this if you're using https://github.com/coreos/prometheus-operator
+  serviceMonitor:
+    enabled: false
+    namespace: monitoring
+    interval: 10s
```

Comment:
Prometheus has a default scrape interval. I suggest not setting overrides by default unless there is a good reason. Having mismatched scrape intervals between different scrape jobs can make it harder to reason about the state of your system

Reply:
Specifying a scraping interval is optional, defaulting to not specified.